FireRender.Maya.Src/python/fireRender/rpr_material_browser.py

```diff
@@ -60,6 +61,8 @@ def __init__(self, *args) :
         # Get the material library path and
         # ensure that it's formatted correctly.5
         libraryPath = self.getLibraryPath()
+        print("ML Log: library path = %" + libraryPath)		
```

Comment:
Should these be logging vs simple print lines?

Reply:
Yes, indeed. Fixed.